pyvista/__init__.py

```diff
@@ -36,11 +36,11 @@
 # get the int type from vtk
 ID_TYPE: type[np.int32 | np.int64] = _get_vtk_id_type()
 
-# determine if using at least vtk 9.0.0
-if vtk_version_info.major < 9:  # pragma: no cover
+# determine if using at least vtk 9.1.0
+if vtk_version_info < (9, 1, 0):  # pragma: no cover
```

Comment:
Isn't support for 9.1 dropped by this PR?
```suggestion
if vtk_version_info < (9, 2, 0):  # pragma: no cover
```